src/assets/scripts/client/navigationLibrary/FixModel.js

```diff
@@ -74,23 +84,41 @@ export default class FixModel extends BaseModel {
         return this._positionModel.relativePosition;
     }
 
+    /**
+     * Facade to access spoken name
+     *
+     * @for FixModel
+     * @return {string}
+     */
+    get spoken() {
+        return this._spoken;
+    }
+
     /**
      * Lifecycle method. Should be run only once on instantiation.
      *
      * @for FixModel
      * @method init
      * @param fixName {string}
+     * @param fixSpoken {string}
      * @param fixCoordinate {array}
      * @param referencePosition {StaticPositionModel}
      * @chainable
      */
-    init(fixName, fixCoordinate, referencePosition) {
+    init(fixName, fixSpoken, fixCoordinate, referencePosition) {
         // TODO: should this be a throwing instead of returning early?
         if (!fixName || !fixCoordinate || !referencePosition) {
             return;
         }
 
         this.name = fixName.toUpperCase();
+
+        if (!fixSpoken) {
+            this._spoken = fixName.toLowerCase();
+        } else {
```

Comment:
Assign the default value before the `if` to avoid the `else`. (comment for myself)

Reply:
This is fixed.